ee/tabby-webserver/src/service/auth.rs

```diff
@@ -219,6 +219,18 @@ impl AuthenticationService for AuthenticationServiceImpl {
         Ok(self.db.get_user_avatar(id.as_rowid()?).await?)
     }
 
+    async fn update_user_name(&self, id: &ID, name: String) -> Result<()> {
+        if is_demo_mode() {
+            bail!("Changing profile data is disabled in demo mode");
+        }
+        if name.is_empty() || name.len() > 100 {
```

Comment:
Not sure if we should limit name lengh this way.

Reply:
1. please move input verification in `update_user_name` in `schema/mod.rs`.
2. please verify with certain standard rules, e.g no space, no special characters (!%$^&...), proper length.